catalog/spatial/geocoding/spatial-geocoding-admin-module/bower.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spatial-admin-module",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "backbone.modelbinder": "v1.0.5",
+    "bootswatch": "v3.2.0",
+    "components-backbone": "1.1.0",
+    "components-bootstrap": "3.2.0",
+    "components-font-awesome": "4.1.0",
+    "icanhandlebarz": "atomicobject/ICanHandlebarz.js#f6ac1d1f8e1d796045563f1483eca6c4911ed7d1",
+    "handlebars": "v1.2.1",
+    "html5shiv": "3.7.2",
+    "jquery": "components/jquery#1.11.0",
+    "jquery-ui": "1.10.4",
+    "lodash": "2.4.1",
+    "marionette": "2.4.1",
+    "moment": "2.5.1",
+    "requirejs": "2.1.14",
+    "require-css": "0.1.5",
+    "requirejs-plugins": "v1.0.2",
+    "lato": "v0.2.1",
+    "jquery-file-upload": "9.5.7",
+    "iframe-resizer": "2.6.2"
```

Comment:
This does not look like the correct set of dependencies for this UI. This looks like a copy and paste of another bower file.